test/include/auxkernels/FunctionGradAux.h

```diff
@@ -0,0 +1,53 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef FUNCTIONGRADAUX_H
+#define FUNCTIONGRADAUX_H
+
+#include "AuxKernel.h"
+
+//Forward Declarations
+class FunctionGradAux;
+class Function;
+
+template<>
+InputParameters validParams<FunctionGradAux>();
+
+/**
+ * Coupled auxiliary gradient
+ */
+class FunctionGradAux : public AuxKernel
+{
+public:
+
+  /**
+   * Factory constructor, takes parameters so that all derived classes can be built using the same
+   * constructor.
+   */
```

Comment:
There is a real information value in this comment. Border-line with cpt. Obvious....